README.md

```diff
@@ -178,82 +220,7 @@ from pydantic import (
-
-```json
-{
-  "name": "./policy.cf",
-  "description": "Local policy file added using cfbs command line",
-  "tags": ["local"],
-  "dependencies": ["autorun"],
-  "steps": ["copy ./policy.cf services/autorun/policy.cf"],
-  "added_by": "cfbs add"
-}
-```
-
-In most cases the steps are designed to copy something locally to the resulting `out/masterfiles` output directory created when running `cfbs build`.
-
-The `source` parameters in each step are relative either to the local cfbs project directory or the `subdirectory` specified in the module definition.
-
-Here is an example from the `client-initiated-reporting` module
-
-```json
-{
-  "name": "client-initiated-reporting",
-  "description": "Enable client initiated reporting and disable pull collection",
-  "tags": ["experimental", "reporting"],
-  "repo": "https://github.com/cfengine/modules",
-  "by": "https://github.com/cfengine",
-  "version": "0.1.1",
-  "commit": "c3b7329b240cf7ad062a0a64ee8b607af2cb912a",
-  "subdirectory": "reporting/client-initiated-reporting",
-  "steps": ["json def.json def.json"],
-  "added_by": "cfbs add"
-}
-```
-
-The source `def.json` to be merged above will be located at `reporting/client-initiated-reporting/def.json`.
-
-During `cfbs build` either the local contents or the repository contents are copied to an auto-generated numeric directory of the form: `out/steps/<step#>_<module_name>_<commit_sha>`.
-This is the working directory where commands are run and where `source` paths should be specified relative to.
-
-```
-out/steps/001_masterfiles_5c7dc5b43088e259a94de4e5a9f17c0ce9781a0f/
-```
-
-The `destination` parameter below should all be relative paths to `out/masterfiles`.
-
-
-### Available steps are:
-
-Note that the build steps are run inside each modules directory (the temporary copy of it inside `out/steps`).
-Scripts and source files are read from that directory.
-Destination is relative to the output policy set (`out/masterfiles`).
-
-#### `copy <source> <destination>`
-- Copy a single file or a directory recursively.
-
-#### `run <command>`
-- Run a shell command / script.
-- Usually used to prepare the module directory, delete files, etc. before a copy step.
-- Running scripts should be avoided if possible.
-
-#### `delete <paths ...>`
-- Delete multiple files or paths recursively.
-
-#### `json <source> <destination>`
-- Merge the source json file into the destination json file.
-
-#### `append <source> <destination>`
-- Append the source file to the end of destination file.
+## The cfbs.json format
 
-#### `directory <source> <destination>`
-- Copy any .cf policy files recursively and add their paths to `def.json`'s `inputs`.
-- Enable `services_autorun_bundles` class in `def.json`.
-- Merge any `def.json` recursively into `out/masterfiles/def.json`.
-- Copy any other files with their existing directory structure to destination.
+More advanced users and module authors may need to understand, write, or edit `cfbs.json` files.
```

Comment:
```suggestion
More advanced users and module authors may need to understand how to write or edit `cfbs.json` files.
```

Reply:
No, the intention here was 3 different activities: understand the JSONs, write them, and edit them.